File: Iniciante/1151.js
let n = lines.shift();

let arr = [0, 1];

for (let i = 2; i < n; i++) {
  arr.push(arr[i-1] + arr[i-2]);
}

for (let i = 0; i < n; i++) {
  if (i === (n - 1)) {
    console.log(arr[i]);
  } else {
    process.stdout.write(arr[i]+ " ");
  }
}

File: Iniciante/1151.py
n = int(input())

arr = [0, 1]

for i in range(2, n):
    arr.append(arr[i-1] + arr[i-2])

for i in range(n):
    if i == (n - 1):
        print(arr[i])
    else:
        print(arr[i], end=" ")
